padinfo/padinfo.py

```diff
@@ -164,12 +164,22 @@ def get_monster_by_no(self, monster_no: int):
         await self._do_id(ctx, query)
 
     @commands.command(name="idna", pass_context=True)
-    async def _do_id_na(self, ctx, *, query: str):
+    async def _do_id_na_multiple_prefixes(self, ctx, *, query: str):

```

Comment:
`For clarity I would leave these as _do_id_[na|all] and change the other to do_id_[na|all]2`

Reply:
Please revert, for a variety of reasons.
1) For style the name of the function should match the name of the command with an _ to indicate private, I used a name override for the command intentionally for a reason I don't recall but dont futz with it.
2) 'multiple prefixes' is a kind of bad suffix for a method name
3) Isn't 'multiple prefixes' wrong under your terminology? This is the old ID, which you're calling 'singular prefix'. The new ID you're working on is the 'multiple prefix' one.

Reply:
all of the names should be fixed now, and i updated to id2na as command name